src/chrome/content/rules/The-ACLU-of-Idaho.xml

```diff
@@ -0,0 +1,14 @@
+<ruleset name="The ACLU of Idaho">
+	<target host="acluidaho.org" />
+	<target host="www.acluidaho.org" />
+
+	<securecookie host="^acluidaho\.org$"
+			name=".+" />
+
+	<test url="http://acluidaho.org/" />
```

Comment:
These test URLs are actually unnecessary: They are generated implicitly by the checker.